apps/studio/components/VyLogo.tsx

```diff
@@ -12,7 +13,7 @@ const Logo = () => {
         fill={mainColor}
       />
       <mask
-        id="a"
+        id={`${id}-a`}
```

Comment:
En liten fiks for å få disse IDene til å bli unike på sider som har flere Vy-logoer. Egentlig ikke noen big deal, men fikser det bare for å få litt mindre støy fra axe-verktøyet vi bruker.